flight/mcu/lib/teensyShared/rk4.h

```diff
@@ -0,0 +1,31 @@
000000049884936905340431429917164534775057003, -0.00000000000020942526189187935999093602609157489081, -0.00000000000414245225601040749310738837216246902712, -0.00000000001789570445050385428595199452049253983216, -0.00000000005636076493535205012399175709398690525126, -0.00000000014604942156233966023751932855324854251822, -0.00000000031173153837571144581390551106514757301325, -0.00000000049889718998833044350278879090895486903001, -0.00000000028766740658974205165675336395576366238513, 0.00000000187736351911293997239389874556302756358761, 0.00000001034453300590629226677060283935694062762423, 0.00000003464191797877778185037238528956593519581020, 0.00000008938858455041137313126358441076058092278345, 0.00000017750172930416809520457442368029932211470623, 0.00000021575720769800536247432561510334325660664945, -0.00000018660145789013343252785474216809413761097858, -0.00000220418853725361969662010705461874238153541228, -0.00000839635618679585508494344486818405925987462979, -0.00002211977951115900022181076645111375000851694494, -0.00004073642334724901861459783747321239388838876039, -0.00003041590215202065892378772982151957648966345005, 0.00013507330564461257800534799144998032716102898121, 0.00075690839307595994892019497513047099346294999123, 0.00223034394552669087974372530425171134993433952332, 0.00411366814645865337685126661426693317480385303497, 0.00190727395441143109111603592964456765912473201752, -0.01921304660195650992893234842995298095047473907471, -0.08581277934908473903341530331090325489640235900879, -0.19550984250010230991456694482621969655156135559082, -0.15241962234822362121100525200745323672890663146973, 0.73715691704171371068099460899247787892818450927734, 3.49009720639244047646343460655771195888519287109375, 6.51769703544439504838692300836555659770965576171875, -2.73789230353590840039146314666140824556350708007812, -50.53373978645147701627138303592801094055175781250000, -116.46571370960862168431049212813377380371093750000000, 51.46970637069863840906691621057689189910888671875000, 871.57435648196644706331426277756690979003906250000000, 1168.72820249389701530162710696458816528320312500000000, -4221.10437662656931934179738163948059082031250000000000, -10902.45880786745874502230435609817504882812500000000000, 26657.65176031351074925623834133148193359375000000000000, 53242.00177372424514032900333404541015625000000000000000, -269330.56008535361615940928459167480468750000000000000000, 473564.28085823717992752790451049804687500000000000000000, -487887.17901733302278444170951843261718750000000000000000, 329743.00213737942976877093315124511718750000000000000000, -151774.68267442920478060841560363769531250000000000000000, 47859.42747184166364604607224464416503906250000000000000, -10165.17644611340620031114667654037475585937500000000000, 1388.46553052871922773192636668682098388671875000000000, -109.45988611012040792047628201544284820556640625000000, 3.55595509400277709488591426634229719638824462890625, 0.53874991700259822202667692181421443819999694824219}};
```

Comment:
poly should be private, maybe move it to a separate file so it doesn't clutter this one

Reply:
@parthshrotri what in the heavens is this lmao? could you explain? maybe I can turn these monstrosities into macros or something

Reply:
nvm i just realized there's like 150 of those. I'll probably just move this to another file

Reply:
its the coefficients for the Cd polyfit, we needed the 150th order to get a close enough fit for it to be accurate.